common/service/ipmi/include/ipmi.h

```diff
@@ -93,148 +93,140 @@ enum {
-	CMD_OEM_1S_GET_SET_GPIO = 0x41,
-	CMD_OEM_1S_GET_FW_SHA256 = 0x43,
-	CMD_OEM_1S_CONTROL_SENSOR_POLLING = 0x45,
-	CMD_OEM_1S_SET_FAN_DUTY_AUTO = 0x50,
-	CMD_OEM_1S_GET_FAN_DUTY = 0x51,
-	CMD_OEM_1S_GET_FAN_RPM = 0x52,
-	// Debug command
-	CMD_OEM_1S_I2C_DEV_SCAN = 0x60,
-
-	CMD_OEM_1S_12V_CYCLE_SLOT = 0x64,
-	CMD_OEM_1S_INFORM_PEER_SLED_CYCLE = 0x66,
-	CMD_OEM_1S_READ_BIC_REGISTER = 0x68,
-	CMD_OEM_1S_WRITE_BIC_REGISTER = 0x69,
-
-	CMD_OEM_1S_PEX_FLASH_READ = 0x72,
-	CMD_OEM_1S_GET_FPGA_USER_CODE = 0x73,
-	CMD_OEM_1S_BRIDGE_I2C_MSG_BY_COMPNT = 0x74,
-
-	CMD_OEM_1S_MULTI_ACCURACY_SENSOR_READING = 0x88,
-	CMD_OEM_1S_GET_BOARD_ID = 0xA0,
-	CMD_OEM_1S_GET_CARD_TYPE = 0xA1,
-	CMD_OEM_1S_NOTIFY_PMIC_ERROR = 0xB0,
+enum { CMD_OEM_1S_MSG_IN = 0x1,
+       CMD_OEM_1S_MSG_OUT = 0x2,
+       CMD_OEM_1S_GET_GPIO = 0x3,
+       CMD_OEM_1S_SET_GPIO = 0x4,
+       CMD_OEM_1S_GET_GPIO_CONFIG = 0x5,
+       CMD_OEM_1S_SET_GPIO_CONFIG = 0x6,
+       CMD_OEM_1S_SEND_INTERRUPT_TO_BMC = 0x7,
+       CMD_OEM_1S_SEND_POST_CODE_TO_BMC = 0x8,
+       CMD_OEM_1S_FW_UPDATE = 0x9,
+       CMD_OEM_1S_GET_FW_VERSION = 0xB,
+       CMD_OEM_1S_GET_BIC_FW_INFO = 0xA,
+       CMD_OEM_1S_GET_POST_CODE = 0x12,
+       CMD_OEM_1S_SET_VR_MONITOR_STATUS = 0x14,
+       CMD_OEM_1S_GET_VR_MONITOR_STATUS = 0x15,
+       CMD_OEM_1S_RESET_BMC = 0x16,
+       CMD_OEM_1S_SET_JTAG_TAP_STA = 0x21,
+       CMD_OEM_1S_JTAG_DATA_SHIFT = 0x22,
+       CMD_OEM_1S_ACCURACY_SENSOR_READING = 0x23,
+       CMD_OEM_1S_CLEAR_CMOS = 0x25,
+       CMD_OEM_1S_ASD_INIT = 0x28,
+       CMD_OEM_1S_PECI_ACCESS = 0x29,
+       CMD_OEM_1S_GET_4BYTE_POST_CODE = 0x2A,
+       CMD_OEM_1S_APML_READ = 0x2C,
+       CMD_OEM_1S_APML_WRITE = 0x2D,
+       CMD_OEM_1S_SEND_APML_REQUEST = 0x2E,
+       CMD_OEM_1S_GET_APML_RESPONSE = 0x2F,
+
+       CMD_OEM_1S_SENSOR_POLL_EN = 0x30,
+       CMD_OEM_1S_GET_BIC_STATUS = 0x31,
+       CMD_OEM_1S_RESET_BIC = 0x32,
+       CMD_OEM_1S_GET_SET_M2 = 0x34,
+       CMD_OEM_1S_SET_SSD_LED = 0x39,
+       CMD_OEM_1S_GET_SSD_STATUS = 0x3A,
+       CMD_OEM_1S_GET_SET_GPIO = 0x41,
+       CMD_OEM_1S_GET_FW_SHA256 = 0x43,
+       CMD_OEM_1S_CONTROL_SENSOR_POLLING = 0x45,
+       CMD_OEM_1S_SET_FAN_DUTY_AUTO = 0x50,
+       CMD_OEM_1S_GET_FAN_DUTY = 0x51,
+       CMD_OEM_1S_GET_FAN_RPM = 0x52,
+       // Debug command
+       CMD_OEM_1S_I2C_DEV_SCAN = 0x60,
+
+       CMD_OEM_1S_12V_CYCLE_SLOT = 0x64,
+       CMD_OEM_1S_INFORM_PEER_SLED_CYCLE = 0x66,
+       CMD_OEM_1S_READ_BIC_REGISTER = 0x68,
+       CMD_OEM_1S_WRITE_BIC_REGISTER = 0x69,
+
+       CMD_OEM_1S_PEX_FLASH_READ = 0x72,
+       CMD_OEM_1S_GET_FPGA_USER_CODE = 0x73,
+       CMD_OEM_1S_BRIDGE_I2C_MSG_BY_COMPNT = 0x74,
+
+       CMD_OEM_1S_MULTI_ACCURACY_SENSOR_READING = 0x88,
+       CMD_OEM_1S_GET_BOARD_ID = 0xA0,
+       CMD_OEM_1S_GET_CARD_TYPE = 0xA1,
+       CMD_OEM_1S_NOTIFY_PMIC_ERROR = 0xB0,
 };
 
-enum {
-	INDEX_SLOT1 = 0x01,
-	INDEX_SLOT3 = 0x03,
+enum { INDEX_SLOT1 = 0x01,
+       INDEX_SLOT3 = 0x03,
```

Comment:
Hi @GoldenBug ,
This format result is follow this command **clang-format command clang-format -style=file -i common/service/ipmi/include/ipmi.h**
And if I change back to original format. It may have clang-format issue like latest commit a17a8a5 in this pr.